CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changes
 
+## Version 2.1.6
```

Comment:
You will have to bump version here also I think https://github.com/simplybusiness/rutabaga/blob/master/lib/rutabaga/version.rb